homeassistant/components/sma/sensor.py

```diff
@@ -84,11 +84,9 @@ def __init__(
     @property
     def name(self) -> str:
         """Return the name of the sensor."""
-        name_prefix = (
-            self._attr_device_info.get("name", "SMA")
-            if self._attr_device_info is not None
-            else "SMA"
-        )
+		name_prefix =  "SMA"
+		if self._attr_device_info is not None:
+			name_prefix = self._attr_device_info.get("name", "SMA")
```

Comment:
Sorry, I had tabs in my suggestion (thanks GitHub :( )

```suggestion
        name_prefix =  "SMA"
        if self._attr_device_info is not None:
            name_prefix = self._attr_device_info.get("name", "SMA")
```